src/py/flwr/common/object_ref.py

```diff
@@ -133,8 +141,10 @@ def load_app(  # pylint: disable= too-many-branches
 
     Note
     ----
-    This function will modify `sys.path` by inserting the provided `project_dir`
-    and removing the previously inserted `project_dir`.
+    - This function will unload all modules in the previously provided `project_dir`,
+      if it is invoked again.
+    - This function will modify `sys.path` by inserting the provided `project_dir`
+      and removing the previously inserted `project_dir`.
     """
     valid, error_msg = validate(module_attribute_str, check_module=False)
```

Comment:
is there a reason why we wouldn't use `check_module=True`?

Reply:
There's no need to have repeated checks here. When passing `check_module=True`, the `validate` function will do similar things as `load_app`.